tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -4,10 +4,33 @@
 import os
 
 from common.layer_test_class import CommonLayerTest
-
 from common.utils.tf_utils import summarize_graph
 
 
+def do_transpose_before(data, use_new_frontend):
```

Comment:
or it is better to name to `transpose_to_nhwc`

Reply:
done